### DISCUSSION TITLE
AWS KMS for Cosign and connaisseur work with keyspec ECC_NIST_P256 but not ECC_NIST_P521

### DISCUSSION THREAD

**Discussion by ZacHaque** [sse-secure-systems/connaisseur#329] [Q&A]
Hi,
I have created KMS key with keyspec ECC_NIST_P521 and was not able to validate from connaisseur running on EKS. It was not able to validate the sign. But changing the key spec to ECC_NIST_P256 only worked. I could not find such limitation stated in the doc.

**Comment by xopham** [comment]
@ZacHaque thanks for reporting :pray: 
Do you get an error or is it just that the signature verification fails? If you get an error what is the `kubectl logs <connaisseur pod>` output?

I suspect you checked that the curve is supported by Cosign right?

**Reply by ZacHaque** [reply]
Thanks for getting back @xopham . Unfortunately, I have not recorded the error, I will re-generate the error and post back here.
